db/migrate/20200411051615_create_merchant_employees.rb

```diff
@@ -0,0 +1,9 @@
+class CreateMerchantEmployees < ActiveRecord::Migration[5.1]
+  def change
+    create_table :merchant_employees do |t|
+      t.references :merchant, foreign_key: true
+      t.references :user, foreign_key: true
+      t.timestamps
+    end
+  end
+end
```

Comment:
name is MerchantEmployees for clarification for us
reference is user because employee is ultimately a user with role 1

Reply:
Is there a way to prevent non-merchant accounts from ever being tied to a merchant in the database?

Reply:
at this point there is not

Reply:
@SMJ289 - standard users shouldn't ever have the ability to modify their role in the database.  we should be okay here without adding any additional logic, methinks.

Reply:
We can look into that in time, but there is no way for non merchants to register as merchants, and since everything is namespaced they shouldn't be able to access any merchant pages at this point where they could get into the database